### PR TITLE
Update mainnet and testnet nodes to v1.3.2, devnet to chain ID 48

### PR DESCRIPTION
**Pull request by clay-aptos**

### CODE REVIEW

**Review by sherry-x — APPROVED** [review]

**Review by saharct — APPROVED** [review]

**Review by thepomeranian — APPROVED** [review]

File: developer-docs-site/docs/releases/devnet-release.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ See the [Latest Aptos Releases](./index.md) index for how to subscribe to GitHub
 
 ## Aptos Framework
 
-Aptos Framework Release commit hash - `1d0dc2c75da1e397effc8bcfa14114652462654d`
+Aptos Framework Release commit hash - `framework-mainnet`
```
Comment (gregnazario) [inline developer-docs-site/docs/releases/devnet-release.md:14]: Need to confirm this, framework hasn't been updated yet

Comment (clay-aptos) [reply developer-docs-site/docs/releases/devnet-release.md:14]: Ack. Converting to Draft until we know. Thanks!

**Comment by clay-aptos** [comment]
Looks like we should rev the CLI too:
https://github.com/aptos-labs/aptos-core/pull/7465

File: developer-docs-site/docs/releases/devnet-release.md (hunk 1 above)
Comment (clay-aptos) [reply developer-docs-site/docs/releases/devnet-release.md:14]: Per the team, we can remove this section and value for the Devnet network.

**Comment by clay-aptos** [comment]
> Looks like we should rev the CLI too: #7465

@gregnazario I updated the CLI on testnet to aptos-cli-v1.0.9. Please let me know if I should do the same in mainnet and devnet. Thanks!

**Comment by clay-aptos** [comment]
> > Looks like we should rev the CLI too: #7465
> 
> @gregnazario I updated the CLI on testnet to aptos-cli-v1.0.9. Please let me know if I should do the same in mainnet and devnet. Thanks!

Nevermind. I forgot we use the same CLI version across networks. Fixed!

**Review by movekevin — APPROVED** [review]

**Pull request merged**